### PR TITLE
Add details page for instance group

### PR DESCRIPTION
**Pull request by nixocio**
This PR also:

* Modify routing system to add `container_groups` as well.
* Add `DetailBadge` component.
* Add container group files as placeholder.

See: https://github.com/ansible/awx/issues/7640
Also: https://github.com/ansible/awx/issues/7467

image

### CODE REVIEW

File: awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupList.jsx [hunk 1]
```diff
@@ -183,16 +198,12 @@ function InstanceGroupList({ i18n }) {
                 key={instanceGroup.id}
                 value={instanceGroup.name}
                 instanceGroup={instanceGroup}
-                detailUrl={`${match.url}/${instanceGroup.id}/details`}
+                detailUrl={verifyDetailUrl(instanceGroup)}
```
Comment (jlmitch5) [inline awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupList.jsx]: I would change this to getDetailUrl, you're not really verifying the url you're returning it.

**Review by jlmitch5 — APPROVED** [review]
looks great other than potentially the small name change!

File: awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupList.jsx (hunk 1 above)
Comment (nixocio) [reply awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupList.jsx]: Updated @jlmitch5. Thanks for your review.

**Comment by jakemcdermott** [comment]
recheck

**Review by dsesami — APPROVED** [review]
Looks great!

**Pull request merged**